Creator criteria (#135)
RedTurtle/redturtle.volto: CHANGES.rst

5.8.1 (unreleased)
------------------

> - Use custom creator criteria to list only creators present in the catalog.

Reviewer: Why do you need to list only the creators in the catalog instead of the original one?

5.8.1 (unreleased)
------------------

> - Use custom creator criteria.

Reviewer: why you need to use a custom creator criteria? It's better to improve infos here because you will forget it in 5 minutes

>       avoiding the plone.many_users setting to send the empty users list.

Reviewer: ```suggestion
      to avoid an empty list when using LDAP and many_users flag
```
